packages/react-components/react-teaching-popover-preview/src/components/TeachingPopoverBody/useTeachingPopoverBodyStyles.styles.ts

```diff
@@ -1,33 +1,62 @@
 import { makeStyles, mergeClasses } from '@griffel/react';
-import type { SlotClassNames } from '@fluentui/react-utilities';
 import type { TeachingPopoverBodySlots, TeachingPopoverBodyState } from './TeachingPopoverBody.types';
+import type { SlotClassNames } from '@fluentui/react-utilities';
 
 export const teachingPopoverBodyClassNames: SlotClassNames<TeachingPopoverBodySlots> = {
   root: 'fui-TeachingPopoverBody',
-  // TODO: add class names for all slots on TeachingPopoverBodySlots.
-  // Should be of the form `<slotName>: 'fui-TeachingPopoverBody__<slotName>`
+  media: 'fui-TeachingPopoverBody__media',
 };
 
-/**
- * Styles for the root slot
- */
 const useStyles = makeStyles({
   root: {
-    // TODO Add default styles for the root element
+    display: 'flex',
+    flexDirection: 'column',
+    paddingBottom: '12px',
   },
-
-  // TODO add additional classes for different states and/or slots
+  media: {
+    width: '288px',
+    objectFit: 'cover',
+    marginBottom: '12px',
+    verticalAlign: 'middle',
+  },
+  mediaShort: { height: '117px' },
```

Comment:
These seem like really specific values (and I'm a bit surpised this one it an odd number). Is it possible to express these heights with relative units?

Reply:
I think the odd number here comes from the expected image ratio based on the enforced 288px width, I'll sync with design on potential for relative units here.

Reply:
We could also consider [`aspect-ratio`](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) which, sadly, is just a bit too new to be fully [supported in our matrix](https://react.fluentui.dev/?path=/docs/concepts-developer-browser-support-matrix--page). Perhaps use `aspect-ratio` with a [fallback to older techniques](https://web.dev/articles/aspect-ratio)?

Reply:
Based on our design guidance, these heights are intended to always be enforced, we use object-fit: contain so that if the aspect ratio wasn't respected it will always fill the space horizontally - due to potential use of async load if we don't enforce this height we may see popover resizing, else forcing this responsibility onto the user.

Reply:
Ah! I guess we can force aspect ratio if the width is enforced, I'll add this.